src/chains/migrations/tests/test_migration_0031.py

```diff
@@ -0,0 +1,28 @@
+from .utils import TestMigrations
+
+
+class Migration0029TestCase(TestMigrations):
+    migrate_from = "0030_wallet"
+    migrate_to = "0031_chain_block_explorer_uri_api_template"
+
+    fixtures = ["src/chains/migrations/tests/fixtures/0030_chains.json"]
+
+    def test_block_explorer_uri_api_template_is_set(self) -> None:
```

Comment:
Just to be sure I completely understand what happens here:
- first you define `from` which and `to` which migration you go.
- The fixture contains pre-migration `json`
- The migration is applied
- Assert that the objects contain the fields according to changes post-migration?

Reply:
That's exactly it 🙌 

Sidenote: Even though we have Factories to generate the models, these factories are tied to latest version of the model. So I took `dumpdata` from migration `0030` and applied the json (instead of creating each object one by one).

Reply:
That makes sense, I suppose that using the factory makes more sense when testing handlers or business logic in general 🤔

Reply:
The `Factory` just creates an object and saves it in the DB so it can be used anywhere – but it is tied to the latest version of the model (which we don't want in this case).